ocis-pkg/oidc/client.go

```diff
@@ -308,7 +308,15 @@ func (c *oidcClient) verifyAccessTokenJWT(token string) (RegClaimsWithSID, jwt.M
 		return claims, mapClaims, err
 	}
 
-	if !claims.VerifyIssuer(c.issuer, true) {
+	validIssuer := false
+	if c.provider.AccessTokenIssuer != "" {
+		// AD FS .well-known/openid-configuration has an optional `access_token_issuer` which takes precedence over `issuer`
+		// See https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-oidce/586de7dd-3385-47c7-93a2-935d9e90441c
+		validIssuer = claims.VerifyIssuer(c.provider.AccessTokenIssuer, true)
+	} else {
+		validIssuer = claims.VerifyIssuer(c.issuer, true)
+	}
+	if !validIssuer {
```

Comment:
```suggestion
	issuer := c.issuer
	if c.provider.AccessTokenIssuer != "" {
		// AD FS .well-known/openid-configuration has an optional `access_token_issuer` which takes precedence over `issuer`
		// See https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-oidce/586de7dd-3385-47c7-93a2-935d9e90441c
		issuer = c.provider.AccessTokenIssuer
	}

	if !claims.VerifyIssuer(issuer, true) {
```

Can we avoid unnecessary `else` statements please? They are nothing but complicated.